src/SIM/SIMbase.h

```diff
@@ -730,6 +734,7 @@ class SIMbase : public SIMinput, public SIMdependency
   IntegrandBase* myProblem; //!< The main integrand of this simulator
   IntegrandMap   myInts;    //!< Set of all integrands involved
   AnaSol*        mySol;     //!< Analytical/Exact solution
+  ModelGenerator* myGen;    //!< Model generator to use
```

Comment:
det er riktig at det strengt tatt ikke trenger å være en member. jeg kan fjerne hvis du vil. jeg tenkte at kanskje kommer det til mer, men det er kanskje å tenke for mye.